### ISSUE TITLE
Channels Buffering 

### ISSUE THREAD

**Issue by Naavi0** [botallen/repository.botallen#357]
- Add-on name: [Jiotv]
- Add-on version: [2.0.0]
- Kodi version: [19.4]
- Operating system version/name: [Android Tv]
- Using from country: [India]

Hello,
Currently Channels are not running smoothly (Buffering)
Before it's worked fine.
If anyone know the solution please let me know.
Thank you.

**Comment by nabakumarchow** [comment]
Same problem . I have the latest version in my tv (nexus 20.0) and i also tried all versions. Too much buffering .

**Comment by MKD1995** [comment]
Guys better stay on matrix 19.5 for now. I hope @iMro0t will update the addon

**Comment by nabakumarchow** [comment]
Buffering on 19.5 also

**Comment by MKD1995** [comment]
For me some channels are buffering sometimes and btw r u using jiotv plugin from this repo ?

https://github.com/Geletinousamigo/plugin.video.jiotv

**Comment by nabakumarchow** [comment]
Yes bro i am using it from 1 week because channels were not playing

**Comment by MKD1995** [comment]
> Yes bro i am using it from 1 week because channels were not playing

Bro install this plugin too from this link and if it's still buffering u can try playing other channel like for me I was watching jio cricket hd a few hours before and it was working fine but when I played it now it was buffering a lot but other channels are working fine.

http://ftp.halifax.rwth-aachen.de/xbmc/addons/matrix/script.module.m3u8/script.module.m3u8-0.5.4+matrix.2.zip

**Comment by nabakumarchow** [comment]
Without kodi is there any way to watch jio tv on tv without mouse

**Comment by snowdog05** [comment]
I have noticed that whenever a cricket match is going on, the other channels buffers a lot. This same happened with football world cup and with ongoing IND vs. SL and IND vs. NZ cricket series. for e.g. there was no buffering yesterday as no match was ongoing, but lots of buffering on Tuesday. This might be issue of Jio servers unable to handle so many requests at same time.

**Comment by drtiwarir** [comment]
It was working perfect in Matrix version of Kodi, I updated to Nexus yesterday and it is not working. I am using it on Nvidia Shield tv.
